inst/extdata/Dockerfile

```diff
@@ -20,12 +20,31 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     python3-pip \
     python3-dev 
 
+RUN apt-get update && apt-get -y upgrade && \
+	apt-get install -y build-essential wget \
+		libncurses5-dev zlib1g-dev libbz2-dev liblzma-dev libcurl3-dev && \
+	apt-get clean && apt-get purge && \
+	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
```

Comment:
I just moved this up so that the image layers are doing the more general / slow steps first. Just might save someone a couple minutes if they need to add a dependency later. https://docs.docker.com/get-started/docker-concepts/building-images/understanding-image-layers/

Reply:
Though looking at it now we are definitely pretty far from the most efficient dockerfile...

Reply:
Yeah, it's okay, we will polish. we're still in the stage of figuring out what is needed here.